src/chol.jl

```diff
@@ -1,9 +1,9 @@
-CholType{T,S<:AbstractMatrix} = Cholesky{T,S}
```

Comment:
This seems to be entirely redundant, so I changed it to use Cholesky directly instead.

Reply:
Moved into #145